### ISSUE TITLE
[igxGrid] Keyboard navigation between cells and filtering row with MCH

### ISSUE THREAD

**Issue by rkaraivanov** [IgniteUI/igniteui-angular#3179]
### Description  
Keyboard navigation between first cell/last filtering chip (and vice versa) does not work in a grid with multi-column headers.

 * igniteui-angular version: 6.2.x+
 * browser: All

### Steps to reproduce  

1. Open the MCH dev demo sample.
2. Select the first cell/last chip in filtering row.
3. Press <kbd>Tab</kbd>/<kbd>Shift + Tab</kbd>

### Result  
No navigation occurs and an error is thrown.

### Expected result  
Navigate to the next focusable element.
